src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -517,7 +517,7 @@
 
 helps['aks update'] = """
     type: command
-    short-summary: Update a managed Kubernetes cluster properties, such as enable/disable cluster-autoscaler
+    short-summary: Update a managed Kubernetes cluster properties, such as enable/disable cluster-autoscaler.  No argumentss reconciles the cluster to its current state.
```

Comment:
Better "No arguments Attempts to move the cluster to its goal state without changing the current cluster configuration. . This operation can be used to move out of a non succeeded state"